app/features/AccountSlice.ts

```diff
@@ -133,6 +202,26 @@ export async function loadAccounts(dispatch: Dispatch) {
     return accounts;
 }
 
+async function loadSimpleViewActive(dispatch: Dispatch) {
+    const simpleViewActive = await window.database.preferences.accountSimpleView.get();
```

Comment:
Why create '~/database/PreferencesDao', when you always use the methods directly?